src/wcm.hpp

```diff
@@ -260,7 +261,7 @@ class OptionSubgroupWidget : public Gtk::Frame
 {
     Gtk::Expander expander;
     Gtk::Box expander_layout = Gtk::Box(Gtk::ORIENTATION_VERTICAL, 10);
-    std::vector<OptionWidget> option_widgets;
+    std::vector<std::unique_ptr<OptionWidget>> option_widgets;
```

Comment:
I wonder why is `std::unique_ptr` necessary (if it is).

Reply:
Yes it is necessary because a sigc::connection can be copied but not moved, whereas some other members can only be moved but not copied. So altogether you can neither copy nor move OptionWidget with this change.